src/pages/dashboardGroupDetail/components/GroupDetail.tsx

```diff
@@ -1,152 +1,102 @@
-      <Grid container item alignContent="center">
-        <Grid item xs={12}>
-          <Typography variant="h6" className="CustomLabelStyle">
-            {t('groupDetail.referents')}
-          </Typography>
-        </Grid>
-        <Grid item xs={12}>
-          <MembersGroup
-            partyGroup={partyGroup}
-            party={party}
-            product={product}
-            isSuspended={isSuspended}
-            productRolesLists={productRolesLists}
-            canEdit={canEdit}
-            onGroupStatusUpdate={onGroupStatusUpdate}
-          />
-        </Grid>
-      </Grid>
       <Grid container item alignContent="center">
         <Grid item xs={3}>
-          <Typography variant="h6" className="CustomLabelStyle">
+          <Typography variant="body2" className={groupLabelClass}>
             {t('groupDetail.creationDate')}
           </Typography>
         </Grid>
-        <Grid item xs={9}>
-          {partyGroup.createdAt ? (
-            <Typography variant="body2" className={groupStatusClass}>
-              {formatDate(partyGroup.createdAt)}
-            </Typography>
+        <Grid item xs={9} display="flex">
+          {partyGroup.createdBy ? (
+            <Box>
+              <Typography variant="body2" className={groupStatusClass} sx={{ fontWeight: '600' }}>
+                {`${partyGroup.createdBy.name} ${partyGroup.createdBy?.surname}`} &nbsp;- &nbsp;
+              </Typography>
+            </Box>
           ) : (
             ''
           )}
-        </Grid>
-        <Grid item xs={3}>
-          <Typography variant="h6" className="CustomLabelStyle">
-            {t('groupDetail.createdByLabel')}
-          </Typography>
-        </Grid>
-        <Grid item xs={9}>
-          {partyGroup.createdBy ? (
-            <Typography variant="body2" className={groupStatusClass}>
-              {`${partyGroup.createdBy.name} ${partyGroup.createdBy?.surname}`}
-            </Typography>
+          {partyGroup.createdAt ? (
+            <Box>
+              <Typography variant="body2" className={groupStatusClass} sx={{ fontWeight: '600' }}>
+                {formatDate(partyGroup.createdAt)}
+              </Typography>
+            </Box>
           ) : (
             ''
           )}
         </Grid>
       </Grid>
       <Grid container item alignContent="center">
         <Grid item xs={3}>
-          <Typography variant="h6" className="CustomLabelStyle">
+          <Typography variant="body2" className={groupLabelClass}>
             {t('groupDetail.modifiedAt')}
           </Typography>
         </Grid>
-        <Grid item xs={9}>
-          {partyGroup.modifiedAt ? (
-            <Typography variant="body2" className={groupStatusClass}>
-              {formatDate(partyGroup.modifiedAt)}
-            </Typography>
+        <Grid item xs={9} display="flex">
+          {partyGroup.modifiedBy ? (
+            <Box>
+              <Typography variant="body2" className={groupStatusClass} sx={{ fontWeight: '600' }}>
```

Comment:
is this fontWeight with fixed value overridable with mui-italia variable?

Reply:
Yes. Modified.